vm/qemu/qemu.go

```diff
@@ -353,6 +353,9 @@ func (pool *Pool) Create(workdir string, index int) (vmimpl.Instance, error) {
 		if i < 1000 && strings.Contains(err.Error(), "Device or resource busy") {
 			continue
 		}
+		if i < 100 && strings.Contains(err.Error(), "Address already in use") {
```

Comment:
Any reason to use a different constant than previous conditions?

Reply:
There's no deep consideration behind the number (as I guess for the two cases above as well). Would you prefer to be it 1000 like the rest?

Reply:
Yes, that's general rule to default to being consistent in such cases.

If the consts are different with no comment, a future reader of the code will wonder why are they different and how this condition is special. Since there is no real answer, trying to find an answer will be unsatisfying waste of time for every reader.

Reply:
Pushed the updated code